NitroxLauncher/Patching/PlatformDetection.cs

```diff
@@ -21,6 +21,14 @@ public static bool IsSteam(string subnauticaPath)
                 return true;
             }
             return false;
+        }
+		public static bool IsMstore(string subnauticaPath)
```

Comment:
Let's be more verbose: `IsMicrosoftStore(...)`

Reply:
alright thought i shorten it but alright thats fine